packages/dev/core/src/Inputs/scene.inputManager.ts

```diff
@@ -824,8 +823,12 @@ export class InputManager {
                 if (!this._meshPickProceed && ((AbstractActionManager && AbstractActionManager.HasTriggers) || scene.onPointerObservable.hasObservers())) {
                     this._initActionManager(null, clickInfo);
                 }
-                if (!pickResult) {
-                    pickResult = this._currentPickResult;
+
+                if (scene.skipPointerUpPicking) {
+                    pickResult = new PickingInfo();
+                } else {
+                    pickResult = scene.pick(this._unTranslatedPointerX, this._unTranslatedPointerY, scene.pointerUpPredicate, false, scene.cameraToUseForPointers);

```

Comment:
After this change, does it mean the code will call scene.pick twice (once for pointer down and once for pointer up) by default?

Reply:
Yes, unless any of the skip bools are set to be true (skipPointerUpPicking is now true by default)